src/reference/environments/example_envs.rst

```diff
@@ -0,0 +1,49 @@
+Example Environments
+====================
+
+A set of possible installations of Cylc:
+
+.. csv-table::
+   :header:  environment, cylc, cylc uiserver, rose
+
+   `Cylc 8 basic`_, ✔️, ,
+   `Cylc 8 with UI Server`_, ✔️,✔️,
+   `Cylc 8 with Rose`_, ✔️,✔️,✔️
+
+Example installation commands:
+
+.. code-block:: bash
+
+   conda env create -f path/to/env.yml --name "name of env"
+
+   # ..or if you haven't specified a python version in the enviroment file:
+   conda env create -f path/to/env.yml --name "name of env" python==<version>
+
+
+Cylc 8 basic
+------------
+
+In this case we are collecting the core workflow engine.
+
+Also demonstrates using different versions of Python.
```

Comment:
"collecting" may be confusing here and there is now just one version of Python documented. Perhaps:

```suggestion
This environment contains the core workflow engine and shows how to specify a particular version of Python.
```
```